web/website/content/faq.md

```diff
@@ -6,11 +6,16 @@ Here are some of the most common questions we hear. Have something else you'd
 like to ask? Pop by our [Discord](https://discord.com/invite/eQcfaCmsNc) and ask
 away!
 
-{{< faq "Cool story Hansel, but what can I actually do with PRQL now?" >}}
+{{< faq "Cool story Hansel, but what can I actually do with PRQL?" >}}
```

Comment:
I did want this to emphasize what it's possible to do *now* — i.e. even if we have plans for greatness, what can we do today? So this is probably the most important word in the question. Happy to rephrase the question / remove the zoolander reference, if it's confusing though.

Reply:
Reverted. I wanted to avoid repetition of "now" later on. I will reword.